libs/platform/src/lib/components/form/radio-group/radio/radio.component.ts

```diff
@@ -22,27 +24,52 @@ let uniqueId = 0;
     templateUrl: './radio.component.html',
     changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class RadioButtonComponent extends BaseInput {
+export class RadioButtonComponent extends BaseInput implements AfterViewInit, FocusableOption {
+    /**
+     * Attached to the aria-label attribute of the host element. In most cases, aria-labelledby will
+     * take precedence so this may be omitted.
+     */
+    @Input('aria-label')
+    ariaLabel = '';
+
+    /**
+     * Users can specify the `aria-labelledby` attribute which will be forwarded to the input element
+     */
+    @Input('aria-labelledby')
+    ariaLabelledby = null;
+
+    /** The 'aria-describedby' attribute is read after the element's label and field type. */
+    @Input('aria-describedby')
+    ariaDescribedby: string;
+
+    /** The 'aria-disabled' for giving accessibility for disabled radio button element. */
+    @Input('aria-disabled')
+    ariaDisabled: boolean;
+
+    /** sets radio button tooltip */
+    @Input()
+    title: string;
+
+    /**
+     * Includes the Radio in the page tab sequence.
+     */
+    @Input()
+    tabIndex = -1;
```

Comment:
Just like my comment in the other PR  .. for the tabindex

Reply:
tabindex for Radio button changes.
for selected radio it should be 0 and for not selected radios, it should be -1.
it will change dynamically once selection of choice changes.
So have to keep it as @Input().
same is done in materials design implementation as well.